auth/views.py

```diff
@@ -11,6 +11,16 @@ class LoginViewSet(TokenObtainPairView):
     permission_classes = [AllowAny]
     serializer_class = LoginSerializer
 
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+
+        try:
+            serializer.is_valid(raise_exception=True)
+        except TokenError as e:
+            raise InvalidToken(e.args[0])
+
+        return Response(serializer.validated_data, status=status.HTTP_200_OK)
+
 class RegisterViewSet(viewsets.ModelViewSet):
```

Comment:
iskreno mozda bih ovu register metodu prebacio u users modul, jer onda treba da kreira samo usera zapravo. Mozda ce nam trebati negde bas odmah user koji je kreiran, a onda cemo morati da kreiramo novu metodu koja ce kreirati usera. I neka ona ipak vraca samog usera, jer ti kada kreiras usera response je 201, a uz to dobijes i usera samog